wallet/src/libtx/aggsig.rs

```diff
@@ -70,6 +70,7 @@ pub fn verify_partial_sig(
 }
 
 /// Just a simple sig, creates its own nonce, etc
+#[allow(deprecated)]
```

Comment:
Those deprecation warnings are actually here on purpose, at least until the underlying secp256k1 API is deemed stable.

Reply:
Yes, I know that purpose. That's why I keep one warning for each function, but suppress the duplicate warning (on same secp256k1 API function).   
So, the remaining 4 warning is enough to remind those secp256k1 API.